storybook/src/components/FieldSet/FieldSet.docs.mdx

```diff
@@ -36,24 +36,25 @@ Add an `aria-describedby` attribute to the input and provide the `id` of Error M
 
 ### Radio group
 
-Use a Field Set to group radio buttons.
-When grouping radio inputs, use `role="radiogroup"` on Field Set to have this grouping explicitly announced as a radio group (the default role is `group`).
-Using `role="radiogroup"` also allows you to use `aria-required` on Field Set, which isn’t allowed for role `group`.
-Always also set `aria-required` on the individual radio buttons though, to make sure it’s read by screen readers.
+Use a Field Set to group several Radios with a legend, description, and / or an error message.
+
+When grouping Radios, add `role="radiogroup"` to the Field Set to have it explicitly announced as a radio group (the default role is just ‘group’).
+The ‘radio group’ role also allows using `aria-required` on Field Set; that isn’t allowed for the ‘group’ role.
+Always also set `aria-required` on the individual Radios though, to make sure it’s read by screen readers.
```

Comment:
‘Always also’ and ‘though’ feel a bit off. Grammarly suggests:

```suggestion
However, always set `aria-required` on the individual Radios to ensure that screen readers can read it.
```

We could replace ‘However’ with ‘Still’ or ‘Even so’.

Reply:
I like 'Even so', it makes it clearer that you also have to set `aria-required` on the individual Radios